packages/cli/src/databases/repositories/authUser.repository.ts

```diff
@@ -1,10 +1,68 @@
 import { Service } from 'typedi';
-import { DataSource, Repository } from '@n8n/typeorm';
+import { DataSource, Repository, In, IsNull, Not } from '@n8n/typeorm';
+import type { DeepPartial, EntityManager, FindOptionsWhere } from '@n8n/typeorm';
+
 import { AuthUser } from '../entities/AuthUser';
+import { Project } from '../entities/Project';
+import { ProjectRelation } from '../entities/ProjectRelation';
 
 @Service()
 export class AuthUserRepository extends Repository<AuthUser> {
 	constructor(dataSource: DataSource) {
 		super(AuthUser, dataSource.manager);
 	}
+
+	/** This returns a user object with all columns, including the ones marked as `select: false` */
```

Comment:
The last part of the comment is confusing me. Where are explicitly selecting selecting all the columns?

Reply:
this is copied from an older version of this branch. I'll update (or delete) the comment.